frontend/javascripts/oxalis/view/right-border-tabs/skeleton_tab_view.js

```diff
@@ -729,27 +737,45 @@ class SkeletonTabView extends React.PureComponent<Props, State> {
                   </AdvancedSearchPopover>
                   <ButtonComponent
                     onClick={this.props.onCreateTree}
-                    title="Create new Tree (C)"
-                    disabled={!this.props.allowUpdate}
+                    title={
+                      isEditingDisabled
+                        ? messages["tracing.read_only_mode_notification"]
+                        : "Create new Tree (C)"
+                    }
+                    disabled={isEditingDisabled}
                   >
                     <i className="fas fa-plus" />
                   </ButtonComponent>
                   <ButtonComponent
                     onClick={this.handleDelete}
-                    title="Delete Selected Trees"
-                    disabled={!this.props.allowUpdate}
+                    title={
+                      isEditingDisabled
+                        ? messages["tracing.read_only_mode_notification"]
+                        : "Delete Selected Trees"
+                    }
+                    disabled={isEditingDisabled}
                   >
                     <i className="far fa-trash-alt" />
                   </ButtonComponent>
                   <ButtonComponent
                     onClick={this.toggleAllTrees}
-                    title="Toggle Visibility of All Trees (1)"
+                    title={
```

Comment:
Since the shortcuts and check boxes to toggle tree visibility still work, these two buttons could remain enabled, I would say. Toggling tree visibility is one of the things that should remain possible in read-only mode in my opinion.